packages/connect-fastify/README.md

```diff
@@ -43,7 +43,7 @@ await server.listen({
 });
 ```
 
-With that server running, you can make requests with any gRPC or Connect client.
+With that server running, you can make requests with any gRPC (-web) or Connect client.
```

Comment:
I feel like `gRPC, gRPC-Web, or Connect` is clearer.  Not a huge deal though.

Reply:
👍